sklearn/ensemble/bagging.py

```diff
@@ -289,6 +289,7 @@ def _fit(self, X, y, max_samples, max_depth=None, sample_weight=None):
 
         # Remap output
         n_samples, self.n_features_ = X.shape
+        self._n_samples = n_samples
```

Comment:
By the way can you please make `max_samples` default to be None in `_fit()` as described in the docstrings?
And add lines below to use `max_samples = self.max_samples` if max_samples is None ?